### PR TITLE
docs: Update limits_and_triggers.adoc with new block limits

### PR DESCRIPTION
**Pull request by odednaor**
### Description of the Changes

New block limits after increasing block size. 

### PR Preview URL

[Current limits](https://starknet-io.github.io/starknet-docs/pr-1188/documentation/tools/limits_and_triggers/)

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1188)

### CODE REVIEW

**Review by iliav-starkware — DISMISSED** [review]

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 1]
```diff
@@ -13,9 +13,9 @@ These are subject to revisions and change on a regular basis
 [%autowidth.stretch]
 |===
 |Entity | Description | Sepolia | Goerli (deprecated) | Mainnet
-|Block time | The average time it takes for a new block to be created on the Starknet blockchain.|3 minutes |3 minutes |3 minutes
+|Block time | If no other limit closes a block, the pending block is closed.| 4 minutes |4 minutes |4 minutes
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:16]:
```suggestion
|Block time | The maximum amount of time within which a pending block is closed, if no other limit is met. | 4 minutes |4 minutes |4 minutes
```

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 2]
```diff
@@ -13,9 +13,9 @@ These are subject to revisions and change on a regular basis
...
 |Block limit (Cairo steps)|Starknet enforces a limit on the number of steps that can be completed
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:17]:
```suggestion
|Block limit (Cairo steps)| The maximum number of Cairo steps that can be completed
```

**Review by stoobie — CHANGES_REQUESTED** [review]

**Review by odednaor — COMMENTED** [review]
Changes lgtm

**Review by stoobie — APPROVED** [review]
LGTM

**Pull request merged**